### PR TITLE
[REF] Implement required fields database changes that have occured du…

### PR DESCRIPTION
**Pull request by seamuslee001**
…ring 5.27.alpha1 development cycle

Overview
----------------------------------------
This implements the database changes made during 5.27.alpha1 to existing sites as the new installs are already done

Before
----------------------------------------
new sites and existing sites differ

After
----------------------------------------
new sites and existing sites unified

ping @eileenmcnaughton @colemanw @totten

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
Given that the test have stymied me at every turn I have confidence that if they pass you have cracked it!

File: CRM/Upgrade/Incremental/php/FiveTwentySeven.php [hunk 1]
```diff
@@ -62,7 +62,40 @@ public function upgrade_5_27_alpha1($rev) {
     $this->addTask('Add serialize column to civicrm_custom_field', 'addColumn',
       'civicrm_custom_field', 'serialize', "int unsigned DEFAULT NULL COMMENT 'Serialization method - a non-null value indicates a multi-valued field.'"
     );
+    $this->addTask('Make the label field required on price field value', 'priceFieldValueLabelRequuired');
+    $this->addTask('Make the name field required on civicrm_membership_type', 'nameMembershipTypeRequuired');
```
Comment (mattwire) [inline CRM/Upgrade/Incremental/php/FiveTwentySeven.php:66]: Trivial, but there is only one "u" in `..Required` :-)

**Comment by mattwire** [comment]
@seamuslee001 I've just removed the merge on pass - would you mind correcting the method names (requuired -> required) and then we can merge on pass again :-)

**Comment by seamuslee001** [comment]
ok done now @mattwire

**Comment by yashodha** [comment]
@seamuslee001 Do we have the related PR for which this upgrade was missing?

**Comment by seamuslee001** [comment]
@yashodha the relevant PRs were https://github.com/civicrm/civicrm-core/pull/17463 https://github.com/civicrm/civicrm-core/pull/17363

**Pull request merged**